Cargo.lock

```diff
@@ -1793,7 +1793,7 @@ checksum = "2b15c43186be67a4fd63bee50d0303afffcef381492ebe2c5d87f324e1b8815c"
 [[package]]
 name = "revm"
 version = "24.0.0"
-source = "git+https://github.com/scroll-tech/revm?branch=feat%2Freth-v74#28f83c2759ac9d2d23969a931cf308b375bb6bd4"
+source = "git+https://github.com/scroll-tech/revm?branch=feat%2Freth-v74-eip-7623#b17c16d296a336ba71460c168ec23167e674be89"
```

Comment:
Need to update to latest commit (or better, the commit after merging the revm PR).